crates/uv-python/fetch-download-metadata.py

```diff
@@ -64,11 +64,10 @@
 # The date at which the default CPython musl builds became dynamically linked
 # instead of statically.
 CPYTHON_MUSL_STATIC_RELEASE_END = 20250311
-# The date at which the linux CPython builds started to have seemingly buggy runtimes
+# The first date at which the linux CPython builds had seemingly buggy runtimes
 CPYTHON_BAD_LINUX_RUNTIME_START = 20250517
-# This alpha was introduced in the bad linux runtime, and we don't want to show it
-# only on some platforms, so hide it everywhere.
-CPYTHON_HIDDEN_ALPHA = "3.14.0a7"
+# The last date at which the linux CPython builds had seemingly buggy runtimes
+CPYTHON_BAD_LINUX_RUNTIME_END = 20250521
```

Comment:
Can we not just drop this entirely? Or do we need it hide some releases?

Reply:
Hmm good point, after getting a clarification from geoffery I *think* we can drop it entirely now. Let me check if anything changes.

Reply:
You were correct, pushed.